assets/js/admin/custom-navigation.js

```diff
@@ -0,0 +1,16 @@
+( () => {
+	const customNavigation = document.querySelector(
+		'#sensei-custom-navigation'
+	);
+	if ( ! customNavigation ) {
+		return;
+	}
+
+	document
+		.querySelector( '#wpbody-content > .wrap' )
+		.prepend( customNavigation );
+	document.querySelector( '.wrap > h1.wp-heading-inline' ).style.display =
+		'none';
+	document.querySelector( '.wrap > a.page-title-action' ).style.display =
```

Comment:
I'm getting an error in the console on the `Course Categories` tab.

<img width="607" alt="Screenshot on 2022-01-24 at 15-53-53" src="https://user-images.githubusercontent.com/1612178/150795458-4237ea12-fcd2-4d6e-87bd-f7bc559a9268.png">

Reply:
Fixed here: https://github.com/Automattic/sensei/pull/4672/commits/da56625b1540435d3c9e8913425e9ce113c5b30a

Reply:
Works! Just out of curiosity, why not use jQuery?

Reply:
I strive not to introduce jQuery if it is possible.
I suppose jQuery does the same under the hood, so there is no need to load an extra library.
The modern JS usually is enough in most cases, from my point of view.